content/api/suppression-list.apib

```diff
@@ -320,104 +255,104 @@ If the X-MSYS-SUBACCOUNT header is not provided, a search will be performed acro
         }
 
 
-### Delete a List Entry [DELETE]
+### Delete a Suppression [DELETE /suppression-list/{recipient}]
 
-Delete a recipient from the list by specifying the recipient's email address in the URI path.
++ Data Structure
+    + type (enum) - The type of suppression to delete. If not provided, the suppression will be deleted for both transactional and non-transactional.
+        + transactional
+        + non_transactional
 
-If the recipient is not in the suppression list, an HTTP status of 404 is returned. If the recipient is in the list, an HTTP status of 204 is returned indicating a successful deletion. Suppression `type` can be specified in the request body. If a type isn't provided, the suppression will be deleted for both transactional and non-transactional.
 
 + Parameters
-    + recipient_email (required, string, `rcpt@example.com`) ... Recipient email address
+    + recipient: `rcpt@example.com` (string, required) - Recipient email address.
 
 + Request
```

Comment:
In this example, can we add a body with a `type` specified? I think that would make the example more helpful.